Compile Warning on the latest Githash

<!-- NOTE: Anything within these brackets will be hidden on the preview of the Issue. -->

* **rAthena Hash**: 2e04ca98b0efb3973e880dc16da57e737d57281a

<!-- Please specify the rAthena [GitHub hash](https://help.github.com/articles/autolinked-references-and-urls/#commit-shas) on which you encountered this issue. 
How to get your GitHub Hash:
1. cd your/rAthena/directory/
2. git rev-parse --short HEAD
3. Copy the resulting hash.
-->

* **Client Date**: 

<!-- Please specify the client date you used. -->

* **Server Mode**: Renewal / Pre-re

<!-- Which mode does your server use: Pre-Renewal or Renewal? -->

* **Description of Issue**: src\map\battle.cpp(3505,6): warning C4554: '&': check operator precedence for possible error; use parentheses to clarify precedence
  * Result: src\map\battle.cpp(3505,6): warning C4554: '&': check operator precedence for possible error; use parentheses to clarify precedence
  * Expected Result: Affected line. if(tsd != nullptr & tsd->bonus.crit_def_rate != 0 && !skill_id && is_attack_critical(wd, src, target, skill_id, skill_lv, false)) {<!-- Describe what you would expect to happen in detail. -->
  * How to Reproduce: <!-- If you have not stated in the description of the result already, please give us a short guide how we can reproduce your issue. -->
  * Official Information:<!-- If possible, provide information from official servers (kRO or other sources) which prove that the result is wrong. Please take into account that iRO (especially iRO Wiki) is not always the same as kRO. -->
  <!-- * _NOTE: Make sure you quote ``` `@atcommands` ``` just like this so that you do not tag uninvolved GitHub users!_ -->

* **Modifications that may affect results**: None
  <!-- * Please provide any information that could influence the expected result. -->
  <!-- * This can be either configurations you changed, database values you changed, or even external source modifications. -->

--- a/src/map/battle.cpp
+++ b/src/map/battle.cpp
@@ -3502,7 +3502,7 @@ static void battle_calc_skill_base_damage(struct Damage* wd, struct block_list *
 				}
 			}
 #ifndef RENEWAL
-			if(tsd != nullptr & tsd->bonus.crit_def_rate != 0 && !skill_id && is_attack_critical(wd, src, target, skill_id, skill_lv, false)) {
+			if(tsd != nullptr && tsd->bonus.crit_def_rate != 0 && !skill_id && is_attack_critical(wd, src, target, skill_id, skill_lv, false)) {
 				ATK_ADDRATE(wd->damage, wd->damage2, -tsd->bonus.crit_def_rate);
 			}
 #endif

